libc/src/__support/CPP/bit.h

```diff
@@ -14,14 +14,13 @@
 #include "src/__support/CPP/limits.h" // numeric_limits
 #include "src/__support/CPP/type_traits.h"
 #include "src/__support/macros/attributes.h"
-#include "src/__support/macros/config.h" // LIBC_HAS_BUILTIN
```

Comment:
This change allows us to remove the dependency on `libc.src.__support.macros.config` from `bit` in libc/src/__support/CPP/CMakeLists.txt, and `:__support_macros_config` from `__support_cpp_bit` in utils/bazel/llvm-project-overlay/libc/BUILD.bazel.

Reply:
Done.